config.php

```diff
@@ -4,8 +4,11 @@
 require_once('constants.php');
 
 
-if (!defined('LDAP_HOST'))
-    define('LDAP_HOST', 'pm-ns.mozilla.org');
+if (!defined('LDAP_HOST')) {
+    header('HTTP/1.0 500 LDAP Error');
```

Comment:
`500` shouldn't be defined as anything other than "Internal Server Error" from spec info I saw, so this shouldn't say "LDAP Error", instead just "Internal Server Error".